README.md

```diff
@@ -205,21 +207,37 @@ if
     and then <condition>)
 then
    <statements>
+elsif
+   <condition>
+then
+   <statements>
+else
+   <statements>
 end if;
 ```
 
 ### If-expressions
 
 ```Ada
-(if <condition> then <expression> else <expression>)
+(if <condition> then <expression> elsif <condition> then <expression> else <expression>)
 ```
 
-If `<condition>` or one of the `<expression>`s do not fit onto one line:
+If the expression or one of the `<condition>`s or `<expression>`s do not fit onto one line:
```

Comment:
Why is expression mentioned twice? Maybe we should simplify that.

Reply:
What I tried to say here is: If the __whole if expression__ or one of __its__ `<condition>`s or `<expression>`s do not fit onto one line:

Reply:
That makes more sense. Thinking about it, wouldn't "if the whole expression does not fit onto one line" suffice?

Reply:
Yes, I think you are right.

Reply:
Fixed by 794c679.